x-pack/plugins/index_management/public/services/extensions_service.ts

```diff
@@ -44,6 +44,8 @@ export interface IndicesListColumn {
   label: string;
   order: number;
   render?: (index: Index) => ReactNode;
+  // return a value used for sorting (only if the value is different from the original value at index[fieldName])
+  sort?: (index: Index) => any;
```

Comment:
I was wondering why we need an additional `sort` property since we could use the already existing `render` property for sorting? After all, the user would expect the column to be sorted by the values that are rendered in this column.

Reply:
That is just to keep the extensions service flexible: there might a property used for render and a property used for sorting, for example `display_size: '58kb'` and `size_in_bytes: 59392`